test/pointing_party/vote_calculator_test.exs

```diff
@@ -0,0 +1,68 @@
+defmodule PointingParty.VoteCalculatorTest do
+  use ExUnit.Case, async: true
+  use ExUnitProperties
+
+  describe "calculate_votes/1" do
+    setup do
+      points_map = StreamData.fixed_map(%{points: integer(1..5)})
+      metas_map  = StreamData.fixed_map(%{
+        metas: list_of(points_map, length: 1),
+      })
+      user = nonempty(map_of(string(:printable), metas_map))
+      [user: user]
+    end
+
+    test "winning value is a list or a integer", %{user: user} do
```

Comment:
I've typically seen `ExUnitProperties` tests use [`property/3`](https://hexdocs.pm/stream_data/ExUnitProperties.html#property/3) instead of `ExUnit.Case.test/3`. What is the effect of using `test` instead?

I'd probably go with `property/3`, and use a helper function to generate a user.